dashboard/app/helpers/levels_helper.rb

```diff
@@ -28,18 +28,15 @@ def url_from_path(path)
   #   using the value from the `data` param.
   def create_channel(data = {}, src = nil)
 
-    result = ChannelsApi.call(request.env.merge(
-      'REQUEST_METHOD' => 'POST',
-      'PATH_INFO' => '/v3/channels',
-      'REQUEST_PATH' => '/v3/channels',
-      'QUERY_STRING' => src ? "src=#{src}" : '',
-      'CONTENT_TYPE' => 'application/json;charset=utf-8',
-      'rack.input' => StringIO.new(data.to_json)
-    ))
-    headers = result[1]
-
-    # Return the newly created channel ID.
-    headers['Location'].split('/').last
+    storage_app = StorageApps.new(storage_id('user'))
```

Comment:
Yep, this is covered by a UI test.  I believe it works because it uses the same underlying `Rack::Request` but I'm not 100% sure.